index.d.ts

```diff
@@ -1,5 +1,13 @@
 declare namespace matcher {
 	interface Options {
+		/**
+		Requires any negated pattern to never match and any normal pattern to match at once. Otherwise, it will be a no-match condition.
+
+		This option may slow down the `matcher.isMatch` with long inputs.
```

Comment:
```suggestion
		This option may slow down `matcher.isMatch` with long inputs.
```

I think it should be explicit about why/ow it might slow down.

Reply:
ok